File: backend/src/main/java/pamiw/eepw/loanmanager/domain/debt/DebtDto.java
package pamiw.eepw.loanmanager.domain.debt;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class DebtDto {
    private Long id;
    private Double totalAmount;
    private Double unpaidAmount;
    private String userEmail;
}

File: frontend/src/types/debt/debtDto.ts
type DebtDto = {
    id: number;
    totalAmount: number;
    unpaidAmount: number;
    userEmail: string;
}

type DebtDtoWithoutId = {
    totalAmount: number;
    unpaidAmount: number;
    userEmail: string;
}

export type {DebtDto, DebtDtoWithoutId};